templates/kubernetes/4/docker-compose.yml

```diff
@@ -52,7 +52,7 @@ etcd:
   environment:
     ETCD_DATA_DIR: /data
     ETCDCTL_ENDPOINT: http://etcd:2379
-    RANCHER_DEBUG: ${DEBUG}
+    RANCHER_DEBUG: true
```

Comment:
It is a good idea to leave this as-is, otherwise it will be impossible to debug any problems that arise. At some point in the future, once etcd template has proven itself reliable, we can turn this off.